src/Component/Blockly/customBlocks.js

```diff
@@ -176,3 +176,133 @@ registerCustom({
     return `!${elem1}===(${elem2})`;
   },
 });
+
+registerCustom({
+  id: 'less_than',
+  definition: {
+    message0: '%1 is less than %2',
+    args0: [
+      { type: 'input_value', name: 'ELEM_1', check: 'Number' },
+      { type: 'input_value', name: 'ELEM_2', check: 'Number' },
+    ],
+    inputsInline: true,
+    output: 'Boolean',
+  },
+  style: 'logic_blocks',
+  generator: (block) => {
+    const firstVar = Blockly.JavaScript.statementToCode(block, 'ELEM_1') || null;
+    const secondVar = Blockly.JavaScript.statementToCode(block, 'ELEM_2') || null;
+    return `${firstVar} < ${secondVar}`;
+  },
+});
+
+registerCustom({
+  id: 'greater_than',
+  definition: {
+    message0: '%1 is greater than %2',
+    args0: [
+      { type: 'input_value', name: 'ELEM_1', check: 'Number' },
+      { type: 'input_value', name: 'ELEM_2', check: 'Number' },
+    ],
+    inputsInline: true,
+    output: 'Boolean',
+  },
+  style: 'logic_blocks',
+  generator: (block) => {
+    const firstVar = Blockly.JavaScript.statementToCode(block, 'ELEM_1') || null;
+    const secondVar = Blockly.JavaScript.statementToCode(block, 'ELEM_2') || null;
+    return `${firstVar} > ${secondVar}`;
+  },
+});
+
+registerCustom({
+  id: 'set_to',
+  definition: {
+    message0: 'set %1 to %2',
+    args0: [
+      { type: 'input_value', name: 'VAR' },
+      { type: 'input_value', name: 'VALUE' },
+    ],
+    inputsInline: true,
+    previousStatement: null,
+    nextStatement: null,
+  },
+  style: 'loop_blocks',
+  generator: (block) => {
+    const variable = Blockly.JavaScript.statementToCode(block, 'VAR') || null;
+    const value = Blockly.JavaScript.statementToCode(block, 'VALUE') || null;
+    return `${variable} = ${value}`;
+  },
+});
+
+registerCustom({
+  id: 'print_var',
+  definition: {
+    message0: 'print %1',
+    args0: [
+      { type: 'input_value', name: 'VARIABLE' },
+    ],
+    inputsInline: true,
+    previousStatement: null,
+    nextStatement: null,
+  },
+  style: 'text_blocks',
+  generator: (block) => {
+    const variable = Blockly.JavaScript.statementToCode(block, 'VARIABLE') || null;
+    return `execution.console.push('${variable}')`;
+  },
+});
+
+registerCustom({
+  id: 'edge_weight',
+  definition: {
+    message0: 'edge weight of %1',
+    args0: [
+      { type: 'input_value', name: 'EDGE' },
+    ],
+    inputsInline: true,
+    output: null,
+  },
+  style: 'loop_blocks',
+  generator: (block) => {
+    const edgeVar = Blockly.JavaScript.statementToCode(block, 'EDGE') || 'edge';
+    return `${edgeVar}.getWeight();\n`;
```

Comment:
Similar to below You need to implement this function to make this block work

Reply:
Also add the new function in the backend. Could you have a look?

Reply:
this requires updates to ./src/graph

Reply:
have added the functions in node.js and edge.js